packages/server/src/index.ts

```diff
@@ -136,35 +138,27 @@ function createServerAdapter<
     if (waitUntilPromises.length > 0) {
       await handleWaitUntils(waitUntilPromises);
     }
-    return response;
```

Comment:
We don't need to return `Response` in `RequestListener`.

Reply:
Yeah, did that to satisfy the typings of `genericRequestHandler`.

Reply:
Yes indeed, the overloads for the `handle` method make this clear now.